src/utils/__tests__/message-test.js

```diff
@@ -70,4 +70,22 @@ describe('shouldBeMuted', () => {
 
     expect(isMuted).toBe(true);
   });
+
+  test('mention narrow messages are never muted', () => {
+    const message = {
```

Comment:
Hmm, these new objects aren't well-typed. Want to get the file type-checked and start using `exampleData` (e.g., `eg.streamMessage`)?

Reply:
I don't really *want* to, but I've gone ahead and done it. It is nice to have this typed :)

Reply:
I can relate! 😅 Thanks for doing this. 🙂